packages/cache/src/cache.ts

```diff
@@ -29,6 +31,15 @@ export class ReserveCacheError extends Error {
   }
 }
 
+function logCacheError(message: string, error: Error): void {
```

Comment:
Revert this change and simply add the conditions where necessary in the code. No need for extracting this into its own function.